0.download_data/scripts/0.download_MNIST_data.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This noteboook download the MNIST dataset
+
+# In[1]:
+
+
+import pathlib
+import pickle
+
+# mnist data parser import
+import mnist
+
+# In[2]:
+
+
+mnist.datasets_url = "https://storage.googleapis.com/cvdf-datasets/mnist/"
+
+# make the mnist directory
+mnist_path = pathlib.Path("../../data/mnist")
+mnist_path.mkdir(parents=True, exist_ok=True)
+
+mnist.temporary_dir = lambda: str(mnist_path)
```

Comment:
Recommend adding a code comment here regarding what this does and why you need it.

Reply:
Yes, I added comments here. I thought this was weird too, it is package-specific for the MNIST parser.